tests/run-pass/fs.rs

```diff
@@ -3,10 +3,28 @@
 
 use std::fs::{File, remove_file};
 use std::io::{Read, Write, ErrorKind};
+use std::path::{PathBuf, Path};
+
+#[cfg(target_os = "linux")]
+fn test_metadata(bytes: &[u8], path: &Path) {
```

Comment:
Could you add a test for a non-existing file, making sure the error is returned correctly?

Reply:
sure.

Reply:
I was afraid that this new test was going to fail. but apparently I took care of this :D

Reply:
Any other kind of test you're afraid might fail? :D

Reply:
Other than someone calling `statx` directly with weird file descriptors. Nah.